src/release/breaking-changes/boxdecoration-shape-migration.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Migration guide for BoxDecoration.circle
+description: Removal of BoxDecoration.circle and how to migrate
+---
+
+## Summary
+
+This change removes `BoxDecoration(shape: BoxShape.circle)`. Existing apps
+should use `ShapeDecoration(shape: CircleBorder())` instead.
```

Comment:
`This change` does not make sense in the context of the website. A lot of users do not follow specific changes on github. 

```suggestion
The `shape` parameter of `BoxDecoration` has been deprecated. Existing uses
of `BoxDecoration(shape: BoxShape.circle)` should use
`ShapeDecoration(shape: CircleBorder())` instead.
```

Reply:
Done :)